elemental-iso-build

```diff
@@ -3,18 +3,18 @@
 set -e
 export DOCKER_BUILDKIT=1
 
-BASEDIR=$(dirname $0)
+BASEDIR=$(dirname $(realpath $0))
 
 build() {
   TEMPDIR=$(mktemp -d)
   mkdir "$TEMPDIR"/iso
   # passing a config file? Copy it to the target that will get copied inside the build
   if [[ -n "$CLOUD_CONFIG_FILE" ]]; then
     echo "Using custom registration from $CLOUD_CONFIG_FILE"
-    cp "$CLOUD_CONFIG_FILE" "$TEMPDIR"/iso/config
+    cp "$CLOUD_CONFIG_FILE" "${TEMPDIR}/iso/config"
   fi
   cp "${BASEDIR}/Dockerfile.iso" "${TEMPDIR}/Dockerfile"
-  curl https://raw.githubusercontent.com/rancher/elemental/main/iso/grub.cfg -fsSL -o "$TEMPDIR"/iso/grub.cfg
+  cp "${BASEDIR}/iso/manifest.yaml" "${TEMPDIR}/iso/manifest.yaml"
```

Comment:
this breaks the whole point of this script, which is for external users to run the script with a config file and get an iso with no cloning of the repo.

Reply:
reverted